### PR TITLE
feat: support maxSuggestions for Input

### PR DESCRIPTION
**Pull request by zlatanpham**

### CODE REVIEW

**Review by tuanddd — APPROVED** [review]

**Comment by tuanddd** [comment]
Should we update the markdown as well?

File: packages/search-ui/src/Input/index.tsx [hunk 1]
```diff
@@ -24,8 +24,7 @@ const Input = React.forwardRef((props: InputProps<any>, ref: React.Ref<HTMLInput
   if (mode === 'suggestions') {
     items = suggestions;
```
Comment (tuanddd) [inline packages/search-ui/src/Input/index.tsx:25]: Do we also need to apply to this mode?

Comment (zlatanpham) [reply packages/search-ui/src/Input/index.tsx]: Nice catch! I've just pushed the update.

Comment (tuanddd) [reply packages/search-ui/src/Input/index.tsx]: We should also update the markdown doc props table. Cheers

**Review by tuanddd — DISMISSED** [review]

**Review by tuanddd — APPROVED** [review]

**Pull request merged**